### ISSUE TITLE
Crash when pasting formula `=YEAR(TODAY())`

### ISSUE THREAD

**Issue by Ameobea** [ironcalc/IronCalc#144]
Pasting this formula:

```
=YEAR(TODAY())
```

into any cell on the web demo causes a crash with this error displayed in the console:

```
index-BxqQcmeI.js:130 Uncaught Error: recursive use of an object detected which would lead to unsafe aliasing in rust
```

I'm not sure if this is just the case of an unsupported formula, but it would definitely be better to return an error rather than putting the entire application into a broken state

**Comment by nhatcher** [comment]
Fixed in #145, will deploy later in the week

**Comment by nhatcher** [comment]
Deployed and regression test added. Feel free to close.
